### PR TITLE
Documentation for JupyterLab

### PR DESCRIPTION
**Pull request by cccs-jc**

### CODE REVIEW

File: example/jupyterlab/README.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+
+# Spark
+
+Sql-language-server now supports a forth `adapter` named `json` which lets you provide your own schema for code completion to the sql-language-server. This is handy for scenarios like Spark where drivers are not easily integrated into a JavaScript process.
+
+The idea is, a user triggers the genration of the schema by calling a function (work in progress). This might be a IPython magic for example
+
+```
+%sql-language-server --generateSchema
```
Comment (joe-re) [inline example/jupyterlab/README.md:113]: @cccs-jc Do you have a plan to work to implement this `generateSchema` command?

Comment (joe-re) [reply example/jupyterlab/README.md:113]: @cccs-jc I understand this generating is gonna be an important thing that this integration goes well.
Also this is gonna be core feature of sql-language-server itself.  If you're busy, I'm also happy to have a plan to implement this. 😄

Comment (cccs-jc) [reply example/jupyterlab/README.md:113]: I have a proof of concept that does the generation of the schema https://github.com/cccs-jc/sparksql_editor

This project is derived from the sparksql_magic project which enables execution of spark SQL statements in a JupyterLab notebook. I have extended it with many more features. The sparksql_editor generates the schema for sql-language-server.

I'm also adding support for TrinoDB and using the same approach where the python magic produces a schema file that the sql-language-server uses for code completion.

sparksql_editor is not finished and I might actually rename it to reflect that it support more than just spark. Other big data SQL engine support should be relatively easy to add to the project.

Comment (joe-re) [reply example/jupyterlab/README.md:113]: @cccs-jc wow, cool. That's brilliant and I'm so excited!

**Review by joe-re — APPROVED** [review]

**Pull request merged**